manifests/c/CiderCollective/Cider/Nightly/1.2.886/CiderCollective.Cider.Nightly.installer.yaml

```diff
@@ -0,0 +1,17 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: CiderCollective.Cider.Nightly
+PackageVersion: 1.2.886
+Installers:
+- MinimumOSVersion: 10.0.0.0
+  Architecture: x86

```

Comment:
```suggestion
  Architecture: x64
```
`x86` boots up a x86 VM so let's try x64 since the application installed fine on x64 for me.

Reply:
from the bot logs
![InstallationVerification-x86-Scope_Undefined-Locale_Undefined-ErrorScreenshot_20220301074006](https://user-images.githubusercontent.com/15158490/156373543-df55a7e2-3811-477d-8b68-8c17ce5dbc89.png)